tests/meta/rsync/test_rsync.py

```diff
@@ -205,7 +206,7 @@ def test_slurm_rsync_single_case(
     sbatch_number: int
     is_complete_delivery: bool
     is_complete_delivery, sbatch_number = rsync_api.slurm_rsync_single_case(
-        case_id=case_obj.internal_id,
+        case=case_obj,
```

Comment:
```suggestion
        case=case,
```

Reply:
There are three fixtures used in the test with "_obj". Would it be good for me to change all of these and reconfigure the tests?